server/lib/actors/hot.go

```diff
@@ -0,0 +1,741 @@
+package actors
+
+import (
+	"log"
+	"fmt"
+	"time"
+	"sort"
+	"sync"
+	"math"
+	"sync/atomic"
+
+	proto "github.com/umegaya/channer/server/proto"
```

Comment:
__[gometalinter]__
* [gotype] could not import github.com/umegaya/channer/server/proto (can't find import: github.com/umegaya/channer/server/proto)